src/models/INeighborLink.ts

```diff
@@ -0,0 +1,8 @@
+import { IRoutePathLink } from '.';
+
+interface INeighborLink {
+    usages: string[];
```

Comment:
usages? What are these and how does one figure it out?

Needs renaming at least

Reply:
nodeUsageByRouteIds: string;